docs/running_psalm/issues/NonInvariantChildProperty.md

```diff
@@ -0,0 +1,26 @@
+# NonInvariantChildProperty
```

Comment:
I'd rename this `NonInvariantPropertyType` as the explanation makes clear it applies to parent-child relationships

Reply:
Thanks. I'll read through your comments now but I need to sleep, so I'll make changes tomorrow.

Reply:
I've renamed it.